cmd/block-generator/generator/server.go

```diff
@@ -0,0 +1,109 @@
+package generator
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/spf13/viper"
+
+	"github.com/algorand/indexer/util"
+)
+
+func initializeConfigFile(configFile string) (config GenerationConfig, err error) {
+	var f *os.File
```

Comment:
nit: An example where named return variables are not useful. You need to declare `f` separately. It's also very easy to create a bug by writing `f, err := os.Open(configFile)`, and you might return a non-`nil` `f` in case of an error.

Reply:
you're right, the var definition isn't necessary because colon-equal does not shadow variables at the top level scope. Fixed.